### DISCUSSION TITLE
Uploading of unlisted versions

### DISCUSSION THREAD

**Discussion by QboiDev** [modrinth/.github#137] [Feature requests]
Uploading unlisted versions would be nice feature for the site.
Like, I can use the unlisted version's link as some sort of "early access" for example.
Well, it's probably not going to be used that often, but sometimes it might.

**Comment by omerien** [comment]
I like it, it would provide a way to "offcially" publish early accesses privately, I know some modmakers might want to publish early accesses only to patreons or things like that
